### ISSUE TITLE
Doesn't this code be used for classify?

### ISSUE THREAD

**Issue by alyato** [brucechou1983/CheXNet-Keras#32]
I check the `train.py`, finding the line`model_train.compile(optimizer=optimizer, loss="binary_crossentropy")`
So i think it's used for detecting.
But the paper also classify 14 diseases?

**Comment by brucechou1983** [comment]
It’s a multitask learning problem.

**Comment by alyato** [comment]
Thanks. I also doesn't understand about the multitask learning.
Then i check the paper.
![1](https://user-images.githubusercontent.com/6815071/38652174-4bea13a2-3e37-11e8-8359-b061471c7738.png)
![2](https://user-images.githubusercontent.com/6815071/38652180-555670e8-3e37-11e8-9299-2d13e9350730.png)

SO does i understand the struct network as following?
![3](https://user-images.githubusercontent.com/6815071/38652317-05b267ee-3e38-11e8-95e2-4fb29addc8cb.jpg)

Are there 14 outputs in the `train.py`,but it's one.
That's why i feel confuse about the line `model_train.compile(optimizer=optimizer, loss="binary_crossentropy")`
Thanks again.

**Comment by alyato** [comment]
Thanks. I m  checking the codes carefully and find my answer.
